epoxy-processor/src/main/java/com/airbnb/epoxy/ClassToGenerateInfo.java

```diff
@@ -66,7 +84,7 @@ public ClassToGenerateInfo(TypeElement originalClassName, ClassName generatedCla
     this.isOriginalClassAbstract = isOriginalClassAbstract;
   }
 
-  private List<ParameterSpec> buildConstructorParamList(List<? extends TypeMirror> params) {
+  private List<ParameterSpec> buildParamList(List<? extends TypeMirror> params) {
```

Comment:
It's nice that we can reuse this method, but I'm thinking if there is only one param we should go with the convention of naming that param after the method. This method was made for constructors so we can't do that here so well. Any ideas on a nice way to have this custom naming for the methods without having to duplicate all this?